openpype/lib/transcoding.py

```diff
@@ -1037,3 +1037,65 @@ def convert_ffprobe_fps_to_float(value):
     if divisor == 0.0:
         return 0.0
     return dividend / divisor
+
+
+def convert_colorspace(
+    input_path,
+    out_filepath,
+    config_path,
+    source_colorspace,
+    target_colorspace,
+    view,
+    display,
+    logger=None
+):
+    """Convert source files from one color space to another.
+
+    Filenames of input files are kept so make sure that output directory
+    is not the same directory as input files have.
+    - This way it can handle gaps and can keep input filenames without handling
+        frame template
+
+    Args:
+        input_path (str): Paths that should be converted. It is expected that
+            contains single file or image sequence of samy type.
+        out_filepath (str): Path to directory where output will be rendered.
+            Must not be same as input's directory.
+        config_path (str): path to OCIO config file
+        source_colorspace (str): ocio valid color space of source files
+        target_colorspace (str): ocio valid target color space
+        view (str): name for viewer space (ocio valid)
+        display (str): name for display-referred reference space (ocio valid)
+        logger (logging.Logger): Logger used for logging.
+    Raises:
+        ValueError: if misconfigured
+    """
+    if logger is None:
+        logger = logging.getLogger(__name__)
+
+    oiio_cmd = [
+        get_oiio_tools_path(),
+        input_path,
+        # Don't add any additional attributes
+        "--nosoftwareattrib",
+        "--colorconfig", config_path,
+        "-o", out_filepath
```

Comment:
Think the output argument needs to be the last in the oiio_cmd list.

Reply:
I'm pretty sure this is the case yes. Code is still invalid.

Reply:
Didn't seem so, but moved nevertheless.